### PR TITLE
[ZH] Skinning

### PR DESCRIPTION
**Pull request by WNCX**
I didn't see any translations in the wiki skin Chinese, so I did the translation actually, hoping to contribute to the encyclodedi of "osu!"

### CODE REVIEW

File: wiki/Skinning/zh.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# 皮肤
+
+皮肤是osu!中的一个主要特色。它能让玩家从默认皮肤中派生制作出自己的皮肤！皮肤可以是有趣的，主题化的，专业的，也可以是几乎无法游玩的。
+
+## 皮肤集
+
+*主页： [皮肤组列表](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**仅限beatmaps，** 如果beatmap外观包含“排名标准”中列出的集合中的单个元素，则它必须包含所述外观集内的所有其他元素。这仅适用于Beatmap皮肤，但其他皮肤爱好者可能也要考虑这些列表。`这段的翻译并不准确！我们希望您能帮助纠错！`
```
Comment (Kotoki1337) [inline wiki/Skinning/zh.md:9]:
```suggestion
**仅限谱面，** 如果谱面外观包含“皮肤组列表”中列出的单个元素，则它必须包含所述皮肤组内的所有其他元素。这仅适用于谱面皮肤，但其他皮肤也需要考虑这些列表。
```

File: wiki/Skinning/zh.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# 皮肤
+
+皮肤是osu!中的一个主要特色。它能让玩家从默认皮肤中派生制作出自己的皮肤！皮肤可以是有趣的，主题化的，专业的，也可以是几乎无法游玩的。
+
+## 皮肤集
```
Comment (Kotoki1337) [inline wiki/Skinning/zh.md:5]:
```suggestion
## 皮肤组
```

File: wiki/Skinning/zh.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## 皮肤元素列表
+
+外观元素列表分为以下几类：
+
+- [外观界面](/wiki/Skinning/Interface)
+- [std皮肤](/wiki/Skinning/osu!)
+- [taiko皮肤](/wiki/Skinning/osu!taiko)
+- [catch皮肤](/wiki/Skinning/osu!catch)
+- [mania皮肤](/wiki/Skinning/osu!mania)
```
Comment (Kotoki1337) [inline wiki/Skinning/zh.md:23]:
```suggestion
- [osu!standard 皮肤](/wiki/Skinning/osu!)
- [osu!taiko 皮肤](/wiki/Skinning/osu!taiko)
- [osu!catch 皮肤](/wiki/Skinning/osu!catch)
- [osu!mania 皮肤](/wiki/Skinning/osu!mania)
```

参考[文章风格指南](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria)，游戏模式请保留全称

File: wiki/Skinning/zh.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- [皮肤音效](/wiki/Skinning/Sounds)
+
+## 也可以参考
+
+- [skin.ini](/wiki/skin.ini)
+- [皮肤变更历史](/wiki/Skinning/History) -旧皮肤元素列表 skinning elements
+- [摘要](https://osu.ppy.sh/community/forums/topics/686664) - 已完成的皮肤元素列表
+- [皮肤常见问题](/wiki/Skinning/FAQ)
+- [指南与重要的线索](/wiki/Skinning/Guides_and_Important_Threads)
```
Comment (Kotoki1337) [inline wiki/Skinning/zh.md:32]:
```suggestion
- [指南与论坛帖子](/wiki/Skinning/Guides_and_Important_Threads)
```

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]
中英之前最好相隔一个空格

File: wiki/Skinning/zh.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- [osu!standard 皮肤](/wiki/Skinning/osu!)
+- [osu!taiko 皮肤](/wiki/Skinning/osu!taiko)
+- [osu!catch 皮肤](/wiki/Skinning/osu!catch)
+- [osu!mania 皮肤](/wiki/Skinning/osu!mania)
+- [皮肤音效](/wiki/Skinning/Sounds)
+
+## 也可以参考
+
+- [skin.ini](/wiki/skin.ini)
+- [皮肤变更历史](/wiki/Skinning/History) -旧皮肤元素列表 skinning elements
```
Comment (kj415j45) [inline wiki/Skinning/zh.md:29]:
```suggestion
- [皮肤变更历史](/wiki/Skinning/History) - 旧皮肤元素列表
```

File: wiki/Skinning/zh.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- [摘要](https://osu.ppy.sh/community/forums/topics/686664) - 已完成的皮肤元素列表
```
Comment (kj415j45) [inline wiki/Skinning/zh.md:30]:
```suggestion
- [汇总](https://osu.ppy.sh/community/forums/topics/686664) - 已完成的皮肤列表
```

File: wiki/Skinning/zh.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- [皮肤常见问题](/wiki/Skinning/FAQ)
+- [指南与论坛帖子](/wiki/Skinning/Guides_and_Important_Threads)
```
Comment (kj415j45) [inline wiki/Skinning/zh.md:32]:
```suggestion
- [指南与重点论坛帖](/wiki/Skinning/Guides_and_Important_Threads)
```

File: wiki/Skinning/zh.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# 皮肤
+
+皮肤是osu!中的一个主要特色。它能让玩家从默认皮肤中派生制作出自己的皮肤！皮肤可以是有趣的，主题化的，专业的，也可以是几乎无法游玩的。
+
+## 皮肤组
+
+*主页： [皮肤组列表](/wiki/Ranking_Criteria/Skin_Set_List)*
+
+**仅限谱面，** 如果谱面外观包含“皮肤组列表”中列出的单个元素，则它必须包含所述皮肤组内的所有其他元素。这仅适用于谱面皮肤，但其他皮肤也需要考虑这些列表。
```
Comment (kj415j45) [inline wiki/Skinning/zh.md:9]:
```suggestion
**仅限谱面**，如果谱面用皮肤包含了“皮肤组列表”中列出的某个元素，则必须同时包含所在皮肤组内的所有其他元素。这仅适用于谱面用皮肤，但其他皮肤作者也可能需要参考这些列表。
```

**Review by kj415j45 — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**